Public/Src/FrontEnd/UnitTests/Script.Ast/Test.BuildXL.FrontEnd.Script.dsc

```diff
@@ -18,8 +18,7 @@ namespace Script {
         assemblyName: "Test.BuildXL.FrontEnd.Script",
         sources: globR(d`.`, "*.cs"),
         runTestArgs: {
-            parallelGroups: categoriesToRunInParallel,
-            weight: 2, //increase weight for frequent timeout pip

```

Comment:
You might want to leave a weight somewhere purely for sake of us exercising the feature in our self-host build